pkg/validate/container_linux.go

```diff
@@ -307,3 +307,201 @@ func createOOMKilledContainer(
 
 	return containerID
 }
+
+var _ = framework.KubeDescribe("Container Mount Readonly", func() {
+	f := framework.NewDefaultCRIFramework()
+
+	var rc internalapi.RuntimeService
+	var ic internalapi.ImageManagerService
+	var runtimeHandler string
+
+	BeforeEach(func() {
+		rc = f.CRIClient.CRIRuntimeClient
+		ic = f.CRIClient.CRIImageClient
+		runtimeHandler = framework.TestContext.RuntimeHandler
```

Comment:
I do not see a need to pass this handler around if it's always `framework.TestContext.RuntimeHandler`

Reply:
fixed